tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp

```diff
@@ -141,6 +162,7 @@ void test_mixed_custom_formattable_type() {
 
 int main() {
     assert(format("{}", basic_custom_formattable_type{"f"}) == "f"s);
+    assert(format("{}", move_only_custom_formattable_type{"f"}) == "f"s);

```

Comment:
Would it be worthwhile to:

* (moderate size) Have test coverage for each family of updated functions? (`format`, `format_to`, `format_to_n`, `formatted_size`, `make_format_args`)
* (large size) Have test coverage for all overloads (`char`/`wchar_t` and plain/`locale`)?

This would have detected `format_to` being missed. However, if writing such test code would be tedious/low-value, I could be persuaded that the current test coverage is sufficient (as each overload didn't need extensive fragile work, just a signature change, combined with larger changes to the common underlying machinery).

Reply:
It's tedious, but not unreasonably so for just the "custom type" tests